src/current/_includes/v23.1/cdc/sink-configuration-detail.md

```diff
@@ -0,0 +1,28 @@
+{{site.data.alerts.callout_danger}}
```

Comment:
In this include files, I have added the general advice from the Webhook sink config section to the Pub/Sub section as well. Can you please confirm that this makes sense?